### PR TITLE
VMware scripting tool (experimental)

### PR DESCRIPTION
**Pull request by thebsdbox**
This brings the VMware scripting tool into InfraKit experimental, example scripts are available in `pkg/x/vmwscript/examples/` for building out a Docker Swarm cluster through the VMware tools.

Signed-off-by: Dan Finneran <daniel.finneran@gmail.com>

### CODE REVIEW

**Comment by stealthybox** [comment]
neat.

**Comment by thebsdbox** [comment]
@chungers Can the ci job be kicked off again, looks like it has failed on the e2e script again?

File: pkg/x/vmwscript/examples/swarm.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        {"name": "Swarm Manager",
+            "note": "Build Swarm leader from Template",
+            "task":{
+               "inputTemplate": "DockerTemplate",
+               "outputName": "manager001",
+               "outputType": "VM",
+               "import":"",
+               "commands": [
+                   {
+                       "type":"execute",
```
Comment (chungers) [inline pkg/x/vmwscript/examples/swarm.json]: Are there types other than `execute`?  I guess a README would be helpful in understanding the design and the schema.  I take it that this executed by the govmomi library?  How does it execute this code on the guest -- does it require SSH or is that through some trusted hypervisor code?

I'd like to understand this more.... maybe we could even leverage some of the patterns here and have a nice integration with the template engine and other parts of infrakit.

File: docs/playbooks/intro/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ If you don't have infrakit or go compiler installed locally, just
 
 ```shell
 
-docker run --rm -v `pwd`:/build infrakit/installer build-infrakit darwin
+docker run --rm -v `pwd`:/build -e GOARCH=amd64 -e GOOS=darwin infrakit/installer build-infrakit
```
Comment (chungers) [inline docs/playbooks/intro/README.md]: This change seems to be from a commit that was merged into master the other day... How come it's showing up in your PR?   Can you rebase?

Comment (thebsdbox) [reply docs/playbooks/intro/README.md]: Doh, tried using the GitHub Desktop tool to make my workflow a little easier and it does some weird syncing/merging of commits.

**Pull request merged**